wdl/src/test/scala/wom/WdlNestedConditionalWomSpec.scala

```diff
@@ -185,4 +186,49 @@ object WdlNestedConditionalWomSpec {
       |    Int? m1_out = m1.out
       |  }
       |}""".stripMargin ++ taskMirror
+
+  val doubleNestedVariableReferences =
+    """|workflow double_nested_var_refs {
+       |
+       |  Int i = 27
+       |  Int j = 32
+       |  Int k = 56
```

Comment:
Curious, are these your lucky lottery numbers?

Reply:
He's spending his weekend writing tests for WOM bugs so I'm guessing not.

Reply:
😢